translator/trace/jaeger/traces_to_jaegerproto.go

```diff
@@ -81,15 +81,15 @@ func resourceSpansToJaegerProto(rs pdata.ResourceSpans) (*model.Batch, error) {
 			continue
 		}
 
-		// TODO: Handle instrumentation library name and version.
+		libraryTags := extractInstrumentationLibraryTags(ils.InstrumentationLibrary())
```

Comment:
maps are more inefficient than slices, consider to return a slice. or simply pass the InstrumentationLibrary to the spanToJaegerProto (will probably be more efficient than maps).